docs/articles_en/openvino_workflow/running_inference_with_openvino/string_tensors.rst

```diff
@@ -0,0 +1,217 @@
+.. {#openvino_docs_OV_UG_string_tensors}
+
+String Tensors
+==============
+
+
+.. meta::
+   :description: Learn how to pass and retrieve text to and from OpenVINO model.
+
+OpenVINO tensor can hold as its elements not only numerical data like floating point or integer numbers,
+but also can contain textual information that is represented as one or multiple strings.
+Such a tensor is called a string tensor.
+
+A string tensor can be passed as an input or retrieved as an output of OpenVINO model.
+Models with string inputs and/or outputs are dedicatedly designed to process textual information.
+For example, string tensors provide a way to feed and retrieve textual data when using OpenVINO
+`tokenizers and detokenizers <https://github.com/openvinotoolkit/openvino_contrib/tree/master/modules/custom_operations/user_ie_extensions/tokenizer/python>`__.
+
+While this section describes basic API to handle string tensors, more practical examples that leverage both
+string tensors and OpenVINO tokenizer can be found in
+`GenAI Samples <https://github.com/openvinotoolkit/openvino.genai/tree/master/text_generation/causal_lm/cpp>`__.
+
+
+Representation
+##############
+
+String tensors are supported in C++ and Python APIs and represented as an instance of `ov::Tensor`
+class with `element_type` parameter equal to `ov::element::string`. Each element of a string tensor is a string
+of arbitrary length, including an empty string, and can be set independently of other elements in the same tensor.
+
+Depending on API used (C++ or Python) underlying data type that represents string when accessing tensor elements is
+different:
+
+ - std::string is used in C++
+
+ - `numpy.str_`/`numpy.bytes_` populated Numpy arrays are used in Python as read-only copy of underlying C++ content
```

Comment:
Also, let us mention standard Python `str`

Reply:
why read-only copy is important here?

Reply:
`str` is not used in tensor representation when accessing elements (the context of that phrase), it can be used as an initializer only